web/src/lambda/locations.js

```diff
@@ -0,0 +1,59 @@
+const Airtable = require('airtable');
```

Comment:
This whole file is not using ES6 syntax because I based off of an example I found online and was too lazy to convert everything.